recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -1,11 +1,16 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
+# Updated patch from meta-suxi for version v2020.04
+#    file://0001-nanopi_neo_air_defconfig-Enable-eMMC-support.patch
+
 SRC_URI_append = " \
+    file://0001-nanopi_neo_air_defconfig-Enable-eMMC-support.patch \
```

Comment:
That patch was already added in meta-sunxi, if the BBMASK will not be set than that line will be not necessery